### ISSUE TITLE
Capture Lists and `--wrapcollections`

### ISSUE THREAD

**Issue by rehatkathuria** [nicklockwood/SwiftFormat#1646]
I have the following:

```
let closure = { @MainActor [foo = state.foo, baz = state.baz] send in ...
```

which, when formatting with `--wrapcollections before-first` gives me the following:

```
let closure = { @MainActor [
					foo = state.foo,
					baz = state.baz,
				] send in
```

which in turn gives me the following error:

`Expected 'weak', 'unowned', or no specifier in capture list`

**Comment by rehatkathuria** [comment]
Looks like this was complaining about the trailing `,` comma generated by another rule

**Comment by nicklockwood** [comment]
The wrapCollections rule is not supposed to wrap capture lists, and trailingCommas shouldn't insert a comma into one either. SwiftFormat was confused by the `@MainActor` attribute and thought the capture list was an array literal. I'll get this fixed.

**Comment by nicklockwood** [comment]
@rehatkathuria fixed in 0.53.5
